src/recipes/modal.ts

```diff
@@ -40,6 +40,7 @@ const modalRecipe = defineSlotRecipe({
         base: 'all 0.4s ease-out',
         _test: 'none',
       },
+      '& p': { margin: '0 0 1em 0' }, // modalDescriptionClass overrides this
```

Comment:
I am okay with this, but I'm uncertain about the value of this comment. Is it uncommon to override styles with more specificity? If we are overriding a style, I think it would be more useful to note that in the place that we override it.

Reply:
Yes, that is true, thanks for bringing it up!